.drone.yml

```diff
@@ -36,6 +36,29 @@ steps:
     event:
     - tag
 
+- name: docker-publish-master
+  image: plugins/docker
+  settings:
+    build_args:
+    - ARCH=amd64
+    - VERSION=${DRONE_BRANCH}-${DRONE_BUILD_NUMBER}-head
```

Comment:
I think `${DRONE_COMMIT_SHA:0:8}` is better than the build number

Reply:
Done, thanks